system/dd/dd_main.c

```diff
@@ -271,6 +276,18 @@ int main(int argc, FAR char **argv)
         }
     }
 
+  if (dd.seek)
+    {
+      ret = lseek(dd.outfd, dd.seek * dd.sectsize, SEEK_SET);
+      if (ret < -1)
```

Comment:
although i guess this is a copy-and-paste from the above dd.skip case,
i don't understand this "ret < -1".
i suspect it should be "ret < 0" or "ret == -1".

Reply:
@dry-75 please update skip case too?

Reply:
Yes c&p  ( on 'minimal change' principle...)
Yes looks like bug.

Yes to update.

Reply:
Update pushed with the above poitned out issue.

Would be good now to focus on actual change, not trying fix other issues in this (very small) feature update.

It is - this change - is usefull however, as I'm exercising it now.